From d8ac23640b0df79188ced9861704cd93e455181b Mon Sep 17 00:00:00 2001
From: Janmm14 <computerjanimaus@yahoo.de>
Date: Mon, 15 Feb 2016 21:28:29 +0100
Subject: Update .gitignore: Add *.orig from merge tools and bungeecord files
 and windows' desktop.ini

---
 .gitignore | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/.gitignore b/.gitignore
index a34de59..f5c6648 100644
--- a/.gitignore
+++ b/.gitignore
@@ -37,3 +37,15 @@ manifest.mf
 
 # delombok
 */src/main/lombok
+
+#Windows
+desktop.ini
+
+#Left merge files
+*.orig
+
+#BungeeCord files
+/config.yml
+/messages.properties
+/plugins/
+/modules/
-- 
1.9.5.msysgit.0

