From c569526dfbbcc0e9c89fe048929b6193628a0ea1 Mon Sep 17 00:00:00 2001
From: Janmm14 <computerjanimaus@yahoo.de>
Date: Mon, 15 Feb 2016 21:28:29 +0100
Subject: Ignore *.orig files which are left by some tools after merging.

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index a34de59..ce24464 100644
--- a/.gitignore
+++ b/.gitignore
@@ -37,3 +37,5 @@ manifest.mf
 
 # delombok
 */src/main/lombok
+
+*.orig
-- 
1.9.5.msysgit.0

