From 5b3a3db7210d649a036409d42fea8a12ff7f3ab5 Mon Sep 17 00:00:00 2001
From: Janmm14 <computerjanimaus@yahoo.de>
Date: Mon, 15 Feb 2016 21:28:29 +0100
Subject: Update .gitignore: Add *.orig from merge tools and bungeecord files
 and windows' desktop.ini

---
 .gitignore | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/.gitignore b/.gitignore
index a34de59..6e23c9b 100644
--- a/.gitignore
+++ b/.gitignore
@@ -37,3 +37,16 @@ manifest.mf
 
 # delombok
 */src/main/lombok
+
+#Windows
+desktop.ini
+
+#Left merge files
+*.orig
+
+#BungeeCord files
+/config.yml
+/messages.properties
+/plugins/
+/modules/
+/locations.yml
-- 
1.9.5.msysgit.0

